client/state/rewind/selectors/get-backup-browser-check-list.ts

```diff
@@ -1,6 +1,14 @@
 import { BackupBrowserItem, BackupBrowserItemCheckList } from 'calypso/state/rewind/browser/types';
 import type { AppState } from 'calypso/types';
 
+const getNodeFullPath = ( node: BackupBrowserItem ): string => {
+	let fullPath = node.ancestors.join( '/' ) + node.path;
```

Comment:
We might need to prepend a `/` before the filename.
```suggestion
	let fullPath = node.ancestors.join( '/' ) + '/' + node.path;
```
<img width="1167" alt="CleanShot 2023-08-31 at 18 24 04@2x" src="https://github.com/Automattic/wp-calypso/assets/1488641/1cca5573-8ebb-4b9f-bd73-a313d35e9b2c">